R/repl.R

```diff
@@ -227,6 +252,37 @@ repl_python <- function(
         return()
       }
 
+      # expand any "!!" as system commands that capture output
+      trimmed <- gsub("!!", "%system ", trimmed)
+
+      # user intends to capture output from system command in var
+      # e.g.:   x = !ls
+      if(grepl("^[[:alnum:]_.]\\s*=\\s*!", trimmed))
+        trimmed <- sub("=\\s*!", "= %system ", trimmed)
+
+      # magic
+      if(grepl("^%", trimmed)) {
```

Comment:
Nit: please use a space after `if` (here and elsewhere).

```suggestion
      if (grepl("^%", trimmed)) {
```

Reply:
Of course! Any chance we could make Cmd+shift+A do this in RStudio? It's nice when I can let the tools do the formatting... (e.g., in python I typically use black)...